tests/integration/models/reddit/test_live_thread.py

```diff
@@ -0,0 +1,26 @@
+"""Test praw.models.LiveThread"""
+from praw.models import LiveThread
+import mock
+import pytest
+
+from ... import IntegrationTest
+
+
+class TestLiveThread(IntegrationTest):
+    @mock.patch('time.sleep', return_value=None)
+    def test_init(self, _):
+        thread_id = 'ukaeu1ik4sw5'
+        data = {'id': thread_id}
+
+        with self.recorder.use_cassette('TestLiveThread_test_init'):
+            thread = LiveThread(self.reddit, thread_id)
+            assert thread.id == thread_id
+            assert thread.title == 'reddit updates'
+
+            thread = LiveThread(self.reddit, _data=data)
+            assert thread.id == thread_id
+            assert thread.title == 'reddit updates'
+
+            with pytest.raises(TypeError):
```

Comment:
Can you add a unit test for this part since it doesn't require any network connectivity?

It would be nice for the LiveThread unit test to contain many of the similar tests as in https://github.com/praw-dev/praw/blob/de1caa50deed07f19c2530f7894e575ec1b88ea9/tests/unit/models/reddit/test_comment.py